From 68ad53b301b6fc722fee6d32a5267c1a4506452d Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Mon, 9 Jul 2012 16:50:22 +1000
Subject: mdmon: fix arg parsing.
Bug-Debian: http://bugs.debian.org/691671
Comment: from stable/bugfix upstream 3.2.6 version
Comment: one of the 3 patches fixing mentioned issues

-t aka --takeover should not be setting container_name.
It sets it to NULL which causes failure when you try
  mdmon --all --takeover

Signed-off-by: NeilBrown <neilb@suse.de>
---
 mdmon.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/mdmon.c b/mdmon.c
index 2093476..dee02a9 100644
--- a/mdmon.c
+++ b/mdmon.c
@@ -306,7 +306,6 @@ int main(int argc, char *argv[])
 			all = 1;
 			break;
 		case 't':
-			container_name = optarg;
 			takeover = 1;
 			break;
 		case OffRootOpt:
-- 
1.7.10.4

